docker/run_jakartamailtck.sh

```diff
@@ -45,7 +49,7 @@ sed -i "s#^SMTP_TO=.*#SMTP_TO=user01@james.local#g" "$TS_HOME/lib/ts.jte"
 mkdir -p ${HOME}/.m2
 
 cd $TS_HOME/tests/mailboxes
-export CLASSPATH=$TS_HOME/tests/mailboxes:$WORKSPACE/jakarta.mail.jar:$WORKSPACE/jakarta.activation.jar:$CLASSPATH
+export CLASSPATH=$TS_HOME/tests/mailboxes:$WORKSPACE/jakarta.mail-api.jar:$WORKSPACE/angus-mail.jar:$WORKSPACE/jakarta.activation.jar:$CLASSPATH
```

Comment:
We need angus-mail because otherwise the next line fails with:

```
Exception in thread "main" java.lang.IllegalStateException: Not provider of jakarta.mail.util.StreamProvider was found
	at jakarta.mail.util.FactoryFinder.find(FactoryFinder.java:64)
	at jakarta.mail.util.StreamProvider.provider(StreamProvider.java:177)
	at jakarta.mail.Session.<init>(Session.java:254)
	at jakarta.mail.Session.getDefaultInstance(Session.java:379)
	at fpopulate.main(fpopulate.java:76)
```

Reply:
> what about activation-api/angus-activation?

I have added them now